perfect_divide.py

```diff
@@ -0,0 +1,17 @@
+"""
+ generator that find a perfect number
+"""
+def pefect_num():
+    n=2
```

Comment:
What is "n"? dont use single letter names for variables - change it please.
Also - if you choose to start from 2 explain why (in a side comment).